README.md

```diff
@@ -17,11 +17,11 @@ When you have completed this code pattern, you will understand how to:
 
 [![](https://bluemix.net/deploy/button.png)](https://console.bluemix.net/developer/appledevelopment/create-project?starterKit=dc43fe0b-f441-3bcb-9c75-2764ece288ce&defaultLanguage=IOS_SWIFT&env_id=ibm%3Ayp%3Aus-south&tenantNavMode=false&defaultDeploymentToolchain=)
 
-Deploy the application automatically using the button above and jump directly to step 4 below.
+Create the application and database automatically using the button above and jump directly to step 4 below.
```

Comment:
Different repo, but I tried a different approach to solve the same issue here: https://github.com/IBM/visual-recognition-ios/blob/master/README.md#steps

Thoughts? cc- @jkingoliver

Reply:
@dolph I like the approach you took in vr-ios